lib/Onnxifi/InlineOnnxifi.cpp

```diff
@@ -0,0 +1,105 @@
+/*
+ * Copyright (c) 2017-present, Facebook, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#include "InlineOnnxifi.h"
+
+#include "glow/Quantization/Quantization.h"
+#include "glow/Quantization/Serialization.h"
+#include "glow/Support/Support.h"
+
```

Comment:
MD5 is really slow. Are you sure you don't need CRC?

Reply:
It's running on the model proto only so it I should be fast compared to other initGraph operations. I just timed `md5sum` on a couple of models and the most was 2ms

Reply:
Okay